spring-cloud-netflix-zuul/src/main/java/org/springframework/cloud/netflix/zuul/filters/route/SimpleHostRoutingFilter.java

```diff
@@ -159,6 +157,14 @@ public void run() {
 					SimpleHostRoutingFilter.this.connectionManager.closeExpiredConnections();
 				}
 			}, 30000, 5000);
+
+			// To support Servlet API 3.1 we need to check if getContentLengthLong exists
+			// Spring 5 minimum support is 3.0, so this stays
+			try {
```

Comment:
I think this should go in the constructors.

Reply:
The reason I put in that in the initialize method was have that in a single place. If you think it's better  having it in the constructors, I could be creating a validation method and calling it from both constructors. Please let me know your thoughts.

Reply:
That's what I would have done. It should be `final` and done in a constructor, calling a method is fine.

Reply:
Fixed.